inc/status.h

```diff
@@ -35,10 +35,12 @@ Status status_get();
 
 // Macros for convenience.
 #define status_code(code) \
-  status_impl_update((code), (__FILE__ ":" STRINGIFY(__LINE__)), (__FUNCTION__), (""))
+  status_impl_update((code), (__FILE__ ":" STRINGIFY(__LINE__)), (__FUNCTION__), "")
 #define status_msg(code, message) \
   status_impl_update((code), (__FILE__ ":" STRINGIFY(__LINE__)), (__FUNCTION__), (message))
+#define status_ok(code) (STATUS_CODE_OK == (code))
+#define status_clear() status_msg(STATUS_CODE_OK, "Clear")
```

Comment:
When we clear the status, we'll probably want to print the previous status code in our debug handler.